Playbooks/Enrich-SentinelIncident-MDATPTVM/azuredeploy.json

```diff
@@ -1,43 +1,25 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
-    "metadata":{
-        "comments": "This playbook will enrich the Client machine that is part of sentinel incident with thread vulnerabilities data (TVM) with CVE that their score is grater then 7.5. Also it automatically add this information to the incident as comments and change the incident severity to High.",
-        "author": "Yaniv Shasha"
-    },
     "parameters": {
-        "PlaybookName": {
+        "workflows_Enrich_SentinelIncident_MDATPTVM_name": {
```

Comment:
Hi @noapocalypse please add "PlaybookName" parameter here, as the parameter name "PlaybookName" is a reserved name and it cannot be replaced with another name. If you use a different parameter name, the playbook may not function properly as Sentinel will not be able to identify it correctly. Thanks!

Reply:
Done